NEWS.md

```diff
@@ -9,7 +9,9 @@
 
 - GitHub repo moved to pharmaverse org and associated broken site links fixed (#803 & #820)
 
-- README and site homepage has been updated with important new section around expectations of admiral, as well as other useful references such as links to conference talks (#868 & #802)
+- README and site homepage has been updated with important new section around expectations of {admiral}, as well as other useful references such as links to conference talks (#868 & #802)
+
+- Addressed typos and appearance of `{admiral}` where identified (#899)
```

Comment:
Is this of interest for the users?

Reply:
probably not. included for full transparency/because the checklist prods me to do so, but typo fixes are likely not newsworthy